brontide/conn.go

```diff
@@ -28,23 +28,44 @@ type Conn struct {
 // A compile-time assertion to ensure that Conn meets the net.Conn interface.
 var _ net.Conn = (*Conn)(nil)
 
+// Dialer provides a way to establish an authenticated connection to a
+// remote address.
+type Dialer interface {
+	Dial(address *lnwire.NetAddress) (*Conn, error)
+}
+
+// NewDialer creates a Dialer for a given private key and
+// connection initiator.
```

Comment:
The lines below should be either be wrapped into those above, or be delimited with a new line.